gluegun.gemspec

```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |spec|
 
   spec.post_install_message = %q{
     Generate sites using:  `./gluegun generate site.yml`
-    For more details see https://gluegun.minio.io
+    For more details see https://gluegun.min.io
```

Comment:
This link doesn't seem to be leading anywhere, we might think of rewording this message @deekoder

Reply:
gluegun.min.io is added to our DNS @kaankabalak